packages/plugin-commands-import/src/import.ts

```diff
@@ -49,6 +71,20 @@ export async function handler (
   await install(await readProjectManifestOnly(opts.dir), installOpts)
```

Comment:
install only accept one workspace, we may use mutateModules to support yarn workspace to pnpm workspace

Reply:
> install only accept one workspace, we may use mutateModules to support yarn workspace to pnpm workspace

Ok, i will think some way to complete the yarn workspace to pnpm workspace transform, maybe i need a judge here.

Reply:
honestly speaking, can u tell some differences between one workspace and more worksapces lock file?
It seems that the yarn will hoist some deps on the top but pnpm not? 
So i just need to deal with deps of the top `package.json` and then to generate the `pnpm-lock.yaml` file?

Reply:
I think it is fine as it is now.